database/src/main/resources/db/migration/V1_49__rental_service_hours_info.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TYPE rental_provider_information
+ DROP ATTRIBUTE "service-hours-info";
```

Comment:
Is someone added data to this column in production? Should we move datas to new column?

Reply:
Good catch, we should check it.

Reply:
There is only one rental service including pick-up-location info. This service does not have service-hours-info. Therefore this migration should be safe.